root/usr/share/nethesis/NethServer/Module/SssdConfig/Wizard/LdapRemoteIp.php

```diff
@@ -76,4 +76,11 @@ public function nextPath()
         return FALSE;
     }
 
-}
\ No newline at end of file
+    public function validate(\Nethgui\Controller\ValidationReportInterface $report)
+    {
+        $this->getValidator('LdapRemoteIpAddress')
+            ->platform('ldap-probe', $this->parameters['LdapRemoteIpAddress'], $this->parameters['LdapRemoteTcpPort']);
```

Comment:
I've checked the Nethgui code :+1: 

https://github.com/NethServer/nethgui/blob/master/Nethgui/Controller/AbstractController.php#L210

A parameter is validated only if it is present in the current request object :)